src/redux/store.ts

```diff
@@ -8,3 +9,7 @@ export const store = configureStore({
   reducer: rootReducer,
   devTools: process.env.NODE_ENV === "development",
 })
+
+export type RootState = ReturnType<typeof store.getState>
```

Comment:
Where `RootState` type is used?

Reply:
While not used anywhere, removed and will be added as needed.

Reply:
Should not be removed, return and use in useSelector